File: cs50/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    int m = n - 1;
    int o = 1;

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < m; j++)
        {
            printf(" ");
        }
        m = m - 1;

        for (int k = 0; k < o; k++)
        {
            printf("#");
        }
        o = o + 1;

        printf("\n");
    }
}

File: cs50/sentimental-mario-less/mario.py
from cs50 import get_int


while True:
    try:
        n = get_int("Height: ")
        if (n >= 1) and (n <= 8):
            break
    except:
        pass

m = n - 1
o = 1

for i in range(n):
    for j in range(m):
        print(" ", end="")
    m -= 1

    for k in range(o):
        print("#", end="")
    o += 1

    print()
